cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,69 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="module")
+def config_manager_rest(config_manager_obj):
+    """Creates provider using REST API."""
+    config_manager_obj.create_rest()
+    assert_response(config_manager_obj.appliance)
+    rest_entity = config_manager_obj.rest_api_entity
+    rest_entity.reload()
+
+    yield rest_entity
+
+    if rest_entity.exists:
+        rest_entity.action.delete()
+        rest_entity.wait_not_exists()
+
+
+@pytest.mark.tier(3)
+@test_requirements.rest
+def test_config_manager_create(config_manager_rest):
```

Comment:
Can we collapse all three of these test cases into a single `test_config_manager_rest_crud()` function please?

Reply:
I was going to do that, but then we need to test delete operation with POST and DELETE, so I had to keep that test case separate, and keeping create and update methods in a single test seemed weird, so I decided to keep these tests separate. I can merge create and update if you think that'd be nicer.